docs/docs/components/guidelines/README.md

```diff
@@ -297,6 +299,10 @@ If the app has a well-supported [Node.js client
 library](../api/#using-npm-packages), that should be preferred to manually
 constructed API requests to reduce code and improve maintenance.
 
+#### Include dependencies in package.json
```

Comment:
I like to surround files with backticks even in headers (VuePress renders them nicely and doesn't include them in the anchor link)